mypy.ini

```diff
@@ -0,0 +1,21 @@
+[mypy]
+pretty = True
+python_version = 3.8
```

Comment:
Is this important? I ask because I foresee people forgetting to update it.

Reply:
Not sure. I think mypy chooses stubs based on [python_version](https://mypy.readthedocs.io/en/stable/config_file.html#confval-python_version).
Forgetting to update this is unlikely to give false negatives: if someone uses a new feature (e.g. python 3.9/3.10: `dict[str, str]` vs `typing.Dict[str,str]`; `str | None` vs `typing.Optional[str]`) it will probably error out.

Reply:
Makes sense